backend/hct_mis_api/apps/reporting/services/generate_report_service.py

```diff
@@ -115,13 +115,14 @@ def format_individual_row(self, individual: Individual) -> tuple:
     def get_households(report: Report) -> QuerySet:
         filter_vars = {
             "business_area": report.business_area,
-            # "program": report.program # TODO Uncomment after add program to household
             "withdrawn": False,
             "last_registration_date__gte": timezone_datetime(report.date_from),
             "last_registration_date__lte": timezone_datetime(report.date_to),
         }
         if report.admin_area.all().exists():
             filter_vars["admin_area__in"] = report.admin_area.all()
+        # if report.program:  # TODO Uncomment after add program to household
```

Comment:
we already have household.program, so it's time to uncomment? :)

Reply:
update